components/list/image/src/index.scss

```diff
@@ -0,0 +1,63 @@
+@import '~@schibstedspain/theme-basic/lib/index';
+@import '~@schibstedspain/sui-image-lazy-load/lib/index';
+
+$m-h-list-image-item-half-gutter: ceil($m-h-list-image-item-full-gutter / 2) !default;
+$m-v-list-image-item-half-gutter: ceil($m-v-list-image-item-full-gutter / 2) !default;
+$m-h-list-image-item-breakpoint-l-gutter: $m-h-list-image-item-full-gutter - 1px !default;
```

Comment:
Isn't there a way to avoid that -1px magic number?

Reply:
For the sui-component, we will keep the horizontal gutter of 16px between cells.

And in our custom theme, we will overwrite this variable to accomplish our specific cell gutter only for screens between 1024 and 1240px (UX requirements).
So, for our project the fixed cell gutter for this screen sizes must be 30px (defined in UX designs as well), that is why we had the half gutter variable of 15px (```$m-h-list-image-item-full-gutter - 1px```), **but we will keep this only in our specific project's theme.**